### PR TITLE
Add some type hints to utils.py

### PR DESCRIPTION
**Pull request by lukebjerring**
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/7987)

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-10-26 20:56:47
Finished: 2017-10-26 21:16:16





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/19955)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/293349264)

File: tools/wpt/utils.py [hunk 1]
```diff
@@ -11,6 +11,13 @@
 
 class Kwargs(dict):
     def set_if_none(self, name, value, err_fn=None, desc=None, extra_cond=None):
+        """
+        :type name: str
```
Comment (jgraham) [inline tools/wpt/utils.py]: This is just documentation, right? Or is some tooling reading it? It doesn't look much like Python 3.5's type hints afaict, but I haven't used those so maybe I'm wrong.

**Review by jgraham — APPROVED** [review]

**Comment by jgraham** [comment]
https://www.python.org/dev/peps/pep-0484/#suggested-syntax-for-python-2-7-and-straddling-code fwiw

**Comment by lukebjerring** [comment]
I've been using Pycharm and was just going off their autogenerated docstring -
 https://www.jetbrains.com/help/pycharm/type-hinting-in-pycharm.html . Will tweak settings to use PEP 484 instead and update this PR.

File: tools/wpt/utils.py (hunk 1 above)
Comment (lukebjerring) [reply tools/wpt/utils.py]: I've been using Pycharm and was just going off their autogenerated docstring -
 https://www.jetbrains.com/help/pycharm/type-hinting-in-pycharm.html . Will tweak settings to use PEP 484 instead and update this PR.

**Review by jgraham — APPROVED** [review]

**Pull request merged**